pwa/app/routes/event.$eventKey.tsx

```diff
@@ -0,0 +1,165 @@
+import { json, LoaderFunctionArgs } from '@remix-run/node';
+import { Link, useLoaderData } from '@remix-run/react';
+import { getEvent, getEventAlliances, getEventMatches } from '~/api/v3';
+import AllianceSelectionTable from '~/components/tba/allianceSelectionTable';
+import InlineIcon from '~/components/tba/inlineIcon';
+import { Badge } from '~/components/ui/badge';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '~/components/ui/tabs';
+import { parseDateString } from '~/lib/utils';
+import BiCalendar from '~icons/bi/calendar';
+import BiGraphUp from '~icons/bi/graph-up';
+import BiInfoCircleFill from '~icons/bi/info-circle-fill';
+import BiLink from '~icons/bi/link';
+import BiListOl from '~icons/bi/list-ol';
+import BiPinMapFill from '~icons/bi/pin-map-fill';
+import BiTrophy from '~icons/bi/trophy';
+import MdiFolderMediaOutline from '~icons/mdi/folder-media-outline';
+import MdiGraphBoxOutline from '~icons/mdi/graph-box-outline';
+import MdiRobot from '~icons/mdi/robot';
+import MdiTournament from '~icons/mdi/tournament';
+
+export async function loader({ params }: LoaderFunctionArgs) {
+  if (params.eventKey === undefined) {
+    throw new Error('Missing eventKey');
+  }
+
+  const event = await getEvent({ eventKey: params.eventKey });
+  const matches = await getEventMatches({ eventKey: params.eventKey });
+  const alliances = await getEventAlliances({ eventKey: params.eventKey });
+
+  return json({ event, matches, alliances });
```

Comment:
How do we handle 404's if the event key provided is not a real event key?

Reply:
We can get to that later imo